Elevel/Domain/Enums/UserRole.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Elevel.Domain.Enums
+{
+    public enum UserRole
+    {
+        Administrator,
+        HumanResourceManager,
```

Comment:
why not just "HR"?

Reply:
I decided that using of meaningfull values is better than abriviature.